config/api.php

```diff
@@ -31,12 +31,24 @@
 
     /*
     |--------------------------------------------------------------------------
-    | Middleware & Authentication
+    | Authentication
+    |--------------------------------------------------------------------------
+    |
+    | By default, the API will be publicly accessible. However, you may define
+    | an API token here which will be used to authenticate requests.
+    |
+    */
+
+    'api_token' => env('STATAMIC_API_TOKEN', null),
```

Comment:
Can you call this `auth_token` instead? We're in the api config so api is implied.

Reply:
Good suggestion - updated!